x/group/simulation/genesis.go

```diff
@@ -0,0 +1,3 @@
+package simulation
+
+// TODO: #270
```

Comment:
If genesis PR #270 is merged before this one, then let's do it as part of this PR, else we'll have to create a follow-up issue for adding genesis simulation.

Reply:
now that #270 is merged, could you address this TODO?